cuda_bindings/docs/source/install.rst

```diff
@@ -0,0 +1,94 @@
+.. SPDX-FileCopyrightText: Copyright (c) 2025 NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+.. SPDX-License-Identifier: LicenseRef-NVIDIA-SOFTWARE-LICENSE
+
+Installation
+============
+
+Runtime Requirements
+--------------------
+
+``cuda.bindings`` supports the same platforms as CUDA. Runtime dependencies are:
+
+* Linux (x86-64, arm64) and Windows (x86-64)
+* Python 3.9 - 3.13
+* Driver: Linux (580.65.06 or later) Windows (580.88 or later)
+* Optionally, NVRTC, nvJitLink, NVVM, and cuFile from CUDA Toolkit 13.x
+
+.. note::
+
+   The optional CUDA Toolkit components are now installed via the ``cuda-toolkit`` metapackage from PyPI for improved dependency resolution. Components can also be installed via Conda, OS-specific package managers, or local installers (as described in the CUDA Toolkit `Windows <https://docs.nvidia.com/cuda/cuda-installation-guide-microsoft-windows/index.html>`_ and `Linux <https://docs.nvidia.com/cuda/cuda-installation-guide-linux/index.html>`_ Installation Guides).
+
+Starting from v12.8.0, ``cuda-python`` becomes a meta package which currently depends only on ``cuda-bindings``; in the future more sub-packages will be added to ``cuda-python``. In the instructions below, we still use ``cuda-python`` as example to serve existing users, but everything is applicable to ``cuda-bindings`` as well.
+
+Installing from PyPI
+--------------------
+
+.. code-block:: console
+
+   $ pip install -U cuda-python
+
+Install all optional dependencies with:
+.. code-block:: console
```

Comment:
The rendering error can be fixed as follows.
```suggestion
Install all optional dependencies with::

.. code-block:: console
```
Remember to introduce code snippets with a double colon, indent the code sample, and include a blank line before and after the code. Please apply this fix to all code blocks.